src/bikeracks/__init__.py

```diff
@@ -42,21 +47,34 @@ def create_app(test_config=None):
         pass
         
     # register close_db and init_db_command with the app
-    from . import db
     db.init_app(app)
     
     # import and register the blueprint from the factory
-    from . import bikes
-    from . import votes
     app.register_blueprint(bikes.bikes)
     app.register_blueprint(votes.votes)
         
     @app.route('/')
     def home(name=None):
         # update map with approved markers
-        # render_template invokes the jinja template engine
         return render_template('base.html', name=name)
         
+    @app.route('/submitFeedback', methods=('POST',))
+    def submitFeedback():
+        csv_file = os.path.join(app.instance_path, "feedback.csv")
+        try:
+            with open(csv_file, 'a') as f:
+                timestamp = int(time())
+                feedback = request.form.get('feedback', '', type=str)
+                row = [timestamp, feedback]
+                # creating a csv writer object 
+                csvwriter = csv.writer(f) 
+                # writing the data rows 
+                csvwriter.writerow(row)
+                return ('OK', 200)
+        except Exception as e:
```

Comment:
Using `try` with a bare `Exception` like this  is _usually_ considered bad style, but since we're using it to get around the fact that we don't have proper access to logs, it's probably fine.

We could fix that by logging to a file, e.g. https://docs.python.org/3/howto/logging-cookbook.html, but that doesn't belong in this PR.

Reply:
Opened an issue.

Reply:
Life is easier for the reviewers if you link to thing you are talking about when you leave a comment like this.

Reply:
#102